pkg/machine/fedora.go

```diff
@@ -59,7 +59,10 @@ func (f FedoraDownload) Get() *Download {
 func (f FedoraDownload) HasUsableCache() (bool, error) {
 	info, err := os.Stat(f.LocalPath)
 	if err != nil {
-		return false, nil
+		if os.IsNotExist(err) {
```

Comment:
use `errors.Is(err, os.ErrNotExist) `

Reply:
I can change but I don't really see why.

Reply:
Look at the godoc comment: https://pkg.go.dev/os#IsNotExist